bolt/lib/Target/X86/X86MCPlusBuilder.cpp

```diff
@@ -211,13 +211,6 @@ class X86MCPlusBuilder : public MCPlusBuilder {
     return false;
   }
 
-  // FIXME: For compatibility with old LLVM only!
-  bool isTerminator(const MCInst &Inst) const override {
-    unsigned Opcode = Inst.getOpcode();
-    return Info->get(Opcode).isTerminator() || X86::isUD1(Opcode) ||
-           X86::isUD2(Opcode);
```

Comment:
Assuming you tested that it's safe to drop ud1/ud2 from terminators.

Reply:
I'm dropping them only for the Linux kernel. For the rest, by default they will still terminate, but the condition check is modified with `isTrap()` check instead of the opcode lookup.